### DISCUSSION TITLE
RunTime ERROR: nVIDIA Driver not installed

### DISCUSSION THREAD

**Discussion by georgestefanz** [lllyasviel/Fooocus#3519] [Q&A]
Just received this error today, totally surprised because till today worked perfectly. 
![image](https://github.com/user-attachments/assets/a940f405-449c-4590-b979-c2a5af878d6e)
Extremely weird, just checked and installed the last driver for my gpu, restarted the pc twice, and still the same error.
What would be the reason?
Help, thanks !

**Comment by Rogala** [comment]
How many video cards are in your system, maybe Fooocus is connecting to the wrong one?

**Comment by robinwang25** [comment]
I also encountered the same problem.
